live-build/includes.chroot/usr/local/bin/compute-system-hash.sh

```diff
@@ -4,16 +4,34 @@ trap '' SIGINT SIGTSTP SIGTERM
 
 set -euo pipefail
 
-echo "TODO: Implement verify hash generation from vx-iso against installed image"
+#hardcoded to nvme for now
+mount -o ro /dev/nvme0n1p1 /mnt
 
-sleep 10
+# Check for the VxLinux-signed.efi and handle if it doesn't exist
+# That aside, we're using the strings command to extract the verity hash
+# from the signed efi binary to later use when calculating the SHV
+if [[ -f /mnt/EFI/debian/VxLinux-signed.efi ]]; then
+  VERITY_HASH=$(strings /mnt/EFI/debian/VxLinux-signed.efi | grep -o verity.hash=[a-zA-Z0-9]* | cut -d'=' -f2)
+else
+  VERITY_HASH=""
+fi
 
-exit 0
+umount /mnt
+
+if [[ ! -z "${VERITY_HASH}" ]]; then
+    if [[ `which xxd` && `which base64` ]]; then
+      base64_hash=$( echo -n ${VERITY_HASH} | xxd -r -p | base64 )
+      echo "Signed Hash: ${base64_hash}"
+      read -p "Press Enter once you have validated the Signed Hash."
+    fi
```

Comment:
Under what circumstances would we expect xxd and base64 to not exist? If they may not exist, let's add an `else` block with a helpful error message, like we do in the lockdown script: https://github.com/votingworks/vxsuite-complete-system/blob/9a05025196ed9a1a0f4f2509232d9e2bf11da59b/config/vendor-functions/lockdown.sh#L141-L149

Right now, seems that we'd just have a silent failure without any indication why

Reply:
In this build, it really isn't expected for them to not exist. I was pulling over the same idea from lockdown, just didn't include the `else` condition since they will be present. I've added the `else` condition, but maybe we just remove it since we know the tools are built into this release?

Reply:
Gotcha, yeah was gonna suggest just removing the if/else if this condition isn't really possible